paddle/fluid/framework/mixed_vector.h

```diff
@@ -410,8 +412,6 @@ class CPUVector : public std::vector<T, std::allocator<T>> {
     return os;
   }
 
-  void resize(size_t size) { this->resize(size); }
```

Comment:
this line cannot compile on mac, it will cause:
```
/Users/qiaolongfei/project/paddle/paddle/fluid/framework/mixed_vector.h:413:28: error: all paths through this function will call itself [-Werror,-Winfinite-recursion]
  void resize(size_t size) { this->resize(size); }
```